public/app/features/alerting/unified/components/rules/RuleDetails.test.tsx

```diff
@@ -42,7 +45,13 @@ const ui = {
   },
 };
 
-const server = setupServer();
+const server = setupServer(
+  http.get(`/api/plugins/${SupportedPlugin.Incident}/settings`, async () => {
```

Comment:
This endpoint wasn't mocked before migration. Why it's necessary now?

Reply:
honestly i have no idea 😅 but without it, there's a console error about trying to call an endpoint that hasn't been mocked

Reply:
Ah, probably we missed that earlier 😅  good catch!